README.md

```diff
@@ -183,7 +176,6 @@ And invoked via `appInsights.<METHOD_NAME>`
 | ----------------------------|------------------------------------------------------------------------------------------------------------|
 | setDistributedTracingMode   | AI only tracing mode is no longer supported. Migrate to using W3C_AND_AI tracing mode. |
 | setAutoCollectHeartbeat     | Heartbeat is not supported in either Azure Monitor OpenTelemetry or the shim.|
-| enableWebInstrumenatation   | WebInstrumentation is not supported in the shim, but is available in Azure Monitor OpenTelemetry as `enableBrowserSdkLoader` |
```

Comment:
Why removing this? I thought you had a mapping for it.

Reply:
I'm removing it from the shim's unsupported list. It should be available for usage.